### PR TITLE
Update helm.md - 2

### PR DESCRIPTION
**Pull request by qingvincentyin**
Refer to the conversation in PR #414.

### CODE REVIEW

File: content/en/docs/use-cases/helm.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ helm install my-traefik traefik/traefik \
 Flux client:
 
 ```sh
-flux create source helm traefik --url https://helm.traefik.io/traefik
+flux create source helm traefik --url https://helm.traefik.io/traefik --namespace traefik
```
Comment (dholbach) [inline content/en/docs/use-cases/helm.md:51]: This is added in line 55.

**Review by dholbach — CHANGES_REQUESTED** [review]
Currently this would add a duplicated flag.

File: content/en/docs/use-cases/helm.md (hunk 1 above)
Comment (dholbach) [reply content/en/docs/use-cases/helm.md:51]: Sorry, I was mistaken.

**Review by dholbach — APPROVED** [review]

**Comment by dholbach** [comment]
@scottrigby and @stefanprodan ACKed the change as well.

**Comment by dholbach** [comment]
Thanks for your PR!

**Review by scottrigby — APPROVED** [review]

**Pull request merged**